internal/provider/log_streams_data_source.go

```diff
@@ -132,6 +132,24 @@ func (d *logStreamsDataSource) Schema(_ context.Context, _ datasource.SchemaRequ
 								},
 							},
 						},
+						"crowdstrike_http_event_collector": schema.SingleNestedAttribute{
+							Description: "Log Stream configuration for the Crowdstrike HTTP Event Collector (HEC) destination type.",
+							Optional:    true,
+							Attributes: map[string]schema.Attribute{
+								"hec_host_port": schema.StringAttribute{
```

Comment:
```suggestion
								"host_port": schema.StringAttribute{
```
Since this is already in a block that references HEC, the prefix is unnecessary.